### PR TITLE
Fix exception message missing format in JDBCLockEngine

### LINKED ISSUE #10191: Throw exception message error
Missing `String.format()` In file [JDBCLockEngine#checkTableLock](https://github.com/apache/shardingsphere/blob/master/shardingsphere-jdbc/shardingsphere-jdbc-core/src/main/java/org/apache/shardingsphere/driver/executor/JDBCLockEngine.java#L106) .

```java
private void checkTableLock(final ShardingSphereLock lock, final Collection<String> tableNames) throws SQLException {
      for (String tableName : tableNames) {
          if (lock.isLocked(LockNameUtil.getTableLockName(DefaultSchema.LOGIC_NAME, tableName))) {
              // Missing format
              throw new SQLException("Table %s is locked");
          }
      }
  }
```

### PR DESCRIPTION
**Pull request by DreamSaddle**
Fixes #10191.

Changes proposed in this pull request:
- Fix exception message missing format in JDBCLockEngine.

### CODE REVIEW

**Comment by RaigorJiang** [comment]
@DreamSaddle  
Hi, DreamSaddle,  please check code style, do not format the code.

File: shardingsphere-jdbc/shardingsphere-jdbc-core/src/main/java/org/apache/shardingsphere/driver/executor/JDBCLockEngine.java [hunk 1]
```diff
@@ -43,34 +43,34 @@
  * JDBC lock engine.
  */
 public final class JDBCLockEngine {
-
```
Comment (RaigorJiang) [inline shardingsphere-jdbc/shardingsphere-jdbc-core/src/main/java/org/apache/shardingsphere/driver/executor/JDBCLockEngine.java:46]: Do not change the indentation.

Comment (DreamSaddle) [reply shardingsphere-jdbc/shardingsphere-jdbc-core/src/main/java/org/apache/shardingsphere/driver/executor/JDBCLockEngine.java:46]: Ok, I will resubmit it.

**Comment by RaigorJiang** [comment]
@DreamSaddle 
Hi, please see `Files changed`, there are still lots of format changes.
image

**Comment by tristaZero** [comment]
@RaigorJiang Nice.
@DreamSaddle Thanks for this PR, hope to see it merged later.

**Comment by DreamSaddle** [comment]
@tristaZero Ok!

**Comment by tristaZero** [comment]
So when will its check style be fixed?

**Comment by DreamSaddle** [comment]
@tristaZero How do I configure code formatting.

**Comment by tristaZero** [comment]
Hi you can import [this setting](https://shardingsphere.apache.org/community/cn/contribute/code-conduct/) if IDEA is your IDE.

**Review by terrymanu — APPROVED** [review]

**Pull request merged**